pandas/tests/scalar/timedelta/test_constructors.py

```diff
@@ -263,6 +263,9 @@ def test_construction_out_of_bounds_td64():
         ("P1W", Timedelta(days=7)),
         ("PT300S", Timedelta(seconds=300)),
         ("P1DT0H0M00000000000S", Timedelta(days=1)),
+        ("PT-6H3M", Timedelta(hours=-6, minutes=3)),
+        ("-PT6H3M", Timedelta(hours=-6, minutes=-3)),
+        ("-PT-6H+3M", Timedelta(hours=6, minutes=-3)),
```

Comment:
try in the invalid test with a single '-' and '-P' which i don't think are valid

Reply:
Added `'P'` and `'-P'`, but not `'-'` since it is not an ISO string (these start with `'P'` or `'-P'`) and therefore not parsed with `parse_iso_format_string`. 

Moreover, on master `pd.Timedelta('-')` does not raises an error and gives back `Timedelta('0 days 00:00:00')`

Reply:
well that's an issue. can you add that test case and fix that as well.

Reply:
Sure, will make another PR for that issue this week.

Reply:
@jreback, created an issue and I am now working on a fix. Do you want to proceed with this PR or do you want to put it on hold till the issue is resolved?